ivy/utils/exceptions.py

```diff
@@ -28,22 +28,22 @@ def _remove_so_log(trace):
         if ".pyx" in repr(st):
             continue
         if "<string>" in repr(st):
-            if "compiled_fn" in repr(st) and module_frame:
+            if "traced_fn" in repr(st) and module_frame:
                 track = True
-                compiled_lineno = st.lineno
+                traced_lineno = st.lineno
 
         if "<module>" in repr(st):
             module_frame = old_frames[idx]
             module_st = st
         elif (
             transpile_frame is None
             and os.path.join("ivy", "compiler") in st.filename
-            and st.name in ["compile", "transpile"]
+            and st.name in ["trace", "transpile"]
```

Comment:
not entirely sure about this one, so just highlighting in case it also needs to be updated only once the .so file is updated

Reply:
yeah, it's comming from `.so` file so it would be compile for now